gunicorn/arbiter.py

```diff
@@ -491,6 +491,8 @@ def murder_workers(self):
         workers = list(self.WORKERS.items())
         for (pid, worker) in workers:
             try:
+                if not worker.booted and time.time() - worker.tmp.last_update() <= self.cfg.startup_timeout:
```

Comment:
It could be maybe a bit more clear here and make this clause not depend on ordering with the one after it.

```python
timeout = self.cfg.timeout if worker.booted else self.cfg.startup_timeout
if time.time() - worker.tmp.last_update() <= timeout:
    continue
```

Is that worth doing?